### PR TITLE
fix(lease): retry up to 5 times to get a lease

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: scripts/lease.sh [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+  local tries=1
+  while [ -z "$TOKEN" ]; do
+    eval $(k8s-claimer --server=k8s-claimer-e2e.deis.com lease create --duration=$CLUSTER_DURATION)
+    if [ -n "$TOKEN" ]; then
+      echo "Leased cluster $CLUSTER_NAME for $CLUSTER_DURATION seconds"
+      echo "TOKEN: $TOKEN"
+      return 0
+    fi
+
+    if [ ${tries} -eq 5 ]; then
```
Comment (krancour) [inline scripts/lease.sh]: Should number of tries be configurable?

Comment (jchauncey) [inline scripts/lease.sh]: k i made it configurable

File: scripts/e2e.sh [hunk 2]
```diff
@@ -29,7 +29,7 @@ return-pod-exit-code() {
 
 wait-for-pod-ready() {
   local name="${1}"
-  local timeout_secs=180
+  local timeout_secs=15
```
Comment (mboersma) [inline scripts/e2e.sh:32]: Is changing `timeout_secs` part of retrying 5 times? This appears to be independent of that change.

Comment (jchauncey) [inline scripts/e2e.sh:32]: I originally at 180 seconds so I could test certain error conditions but it shoudl be at 15 seconds. Just seemed innocuous to go ahead and change this back to what it should be with this pr.

**Comment by sgoings** [comment]
Removing 2.0-rc2 milestone assignment as a way to keep our rc2 milestone clean and customer deliverable focused: https://github.com/pulls?utf8=%E2%9C%93&q=user%3Adeis+milestone%3Av2.0-rc2+

**Pull request merged**